pkg/resource/aws/aws_iam_access_key_ext.go

```diff
@@ -14,3 +20,10 @@ func (r *AwsIamAccessKey) NormalizeForState() (resource.Resource, error) {
 func (r *AwsIamAccessKey) NormalizeForProvider() (resource.Resource, error) {
 	return r, nil
 }
+
+func (r *AwsIamAccessKey) String() string {
+	if aws.StringValue(r.User) == "" {
```

Comment:
I think checking for `r.User == nil` is enough there, and thus it will be more consistent whit what you did before on route53 resources.

Reply:
Done.